dev-util/statifier/statifier-1.7.4.ebuild

```diff
@@ -20,6 +22,13 @@ src_prepare() {
 	# Respect users CFLAGS and LDFLAGS
 	sed -i -e 's/-Wall -O2/$(CFLAGS) $(LDFLAGS)/g' src/Makefile || die
 
+	# Don't compile 32-bit on no-multilib profiles
```

Comment:
Would be better to use our user-facing multilib stuffs. In this case I'm going to guess this tool is not very flexible, so you may want to use `multilib-build`, declare `MULTILIB_COMPAT=( abi_x86_{32,64} )` and test for `use abi_x86_32`.

Reply:
Thanks, I've pushed a new version.